backend/route/vehicule.js

```diff
@@ -56,6 +75,7 @@ router.get("/:id/historique", (req, res) => {
       } else {
         // Si tout s'est bien passé, on envoie le résultat de la requête SQL en tant que JSON.
         res.json(results);
+
```

Comment:
(This empty line makes me think maybe Prettier is not properly used on your machine. If you need help to check it, feel free to poke me)

Reply:
The request wasn't closed, so prettier didn't worked. But don't know why braces were missing, maybe due to a merge with dev ?